### PR TITLE
Ability to run production check from the front cli

### PR DESCRIPTION
**Pull request by spolu**
## Description

front/cli <3 production-checks

## Risk

None

## Deploy Plan

N/A

### CODE REVIEW

File: front/admin/cli.ts [hunk 1]
```diff
@@ -477,16 +478,51 @@ const registry = async (command: string) => {
   }
 };
 
+const productionCheck = async (command: string, args: parseArgs.ParsedArgs) => {
+  switch (command) {
+    case "run": {
+      if (!args.check) {
+        throw new Error("Missing --check argument");
```
Comment (Fraggle) [inline front/admin/cli.ts:485]: nit: couldn't that run all checks ?

File: front/admin/cli.ts [hunk 2]
```diff
@@ -508,9 +544,13 @@ const main = async () => {
...
     case "registry":
       return registry(command);
+    case "production-check":
+      return productionCheck(command, argv);
     default:
       console.log(
-        "Unknown object type, possible values: `workspace`, `user`, `data-source`, `event-schema`, `conversation`, `transcripts`"
+        "Unknown object type, possible values: " +
+          "`workspace`, `user`, `data-source`, `event-schema`, `conversation`, " +
+          "`transcripts`, `registry`, `production-check`"
```
Comment (Fraggle) [inline front/admin/cli.ts:553]: nit: maybe it's an opportunity to type objectType (so the error message could be dynamically generated and we could assertNever in the switch case).

**Review by Fraggle — APPROVED** [review]
LGTM.

File: front/admin/cli.ts (hunk 2 above)
Comment (spolu) [reply front/admin/cli.ts:553]: Will give it 1mn and otherwise move forward :)

Comment (spolu) [reply front/admin/cli.ts]: (done)

**Pull request merged**